src/napistu/gcs/downloads.py

```diff
@@ -14,14 +15,15 @@
 logger = logging.getLogger(__name__)
 
 
-def load_public_cpr_asset(
+def load_public_napistu_asset(
```

Comment:
While I think is convenient here to have some default assets hardcoded, I think it would be good to make sure that there is an easy mechanism to overwrite this.
Currently this would require to hot-patch the napistu library to change GCS_ASSETS.

A good way may be to make sure that functions like `load_public_napistu_asset` have a `gcs_asset` argument, that defaults to GCS_ASSETS if not set explicitly.

This would make it straight forward to overwrite this and also makes it easier for a user to find where the data comes from (because they would see that `constants.GCS_ASSETS ` is used as a default).